src/jquery.mockjax.js

```diff
@@ -275,9 +275,17 @@
 						} else {
 							this.responseText = mockHandler.responseText;
 						}
-						if( typeof mockHandler.status === 'number' || typeof mockHandler.status === 'string' ) {
+						
+						if ($.isArray(mockHandler.status)) {
+							var min = Math.ceil(0);
+							var max = Math.floor(mockHandler.status.length-1);
+							// get a random integer between two values, inclusive
+							var idxStatus = Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
I think you've over complicated this since `min` is always `0` now. Could be just `Math.floor(Math.random() * mockHandler.status)`

(You do a -1 up above in the `max` calculation, then a +1 down here...)